src/styles/less/Hud.less

```diff
@@ -39,6 +39,7 @@
 
   @media screen and (max-width: 500px) {
     margin: 0 auto;
+    padding: 25px 0;
```

Comment:
I like that you added padding, but I would play around with a small size. Somewhere between 5px - 10px would still help with legibility without causing unnecessary space. Since this will be on a phone, the less space we waste, the less the user will have to scroll. However, this is just my opinion and doesn't need to be done.